cmd/kubectl-gadget/main.go

```diff
@@ -79,7 +79,7 @@ func main() {
 		Use:   "update-catalog",
```

Comment:
I think it can be misleading for users to have `update-deploy-info`. The main purpose of this command is to update the gadgets available on kubectl-gadget, so I think we can continue to use update-catalog there. @flyth thoughts?

Reply:
Yeah, agree - `update-info` would IMHO just make it more abstract. If we want to make the command a bit more generic, maybe `sync` would be an option  (helptext: "synchronize gadget information with your cluster" or something like that). Then the user wouldn't have to know what a "catalog" or "deploy info" is.

Reply:
Makes sense. Updated it to be `sync`.